app/controllers/relationships_controller.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+class RelationshipsController < ApplicationController
+  before_action :check_logged_in
+
+  # POST /users/:user_id/relationships
+  def create
+    user = User.find(params[:user_id])
+
+    if current_user.follow(user)
```

Comment:
simple

一時変数に入れる必要がなさそう。
```suggestion
    if current_user.follow(User.find(params[:user_id]))
```

Reply:
対応しました。
https://github.com/ok-os-job-change-team/tetsuya-twitter-clone-bootcamp/pull/74/commits/c53016707e6fd8d3f29bdc82ffac8d8fd1c3f2a4